scripts/build_example

```diff
@@ -44,56 +40,19 @@ else
     TARGET="$NAME"
 fi
 
-for module in examples/"${TARGET}"/module*/rust/Cargo.toml; do
+for module in examples/"$TARGET"/module*/rust/Cargo.toml; do
```

Comment:
(Aside: the Google shell style guide suggests ['Prefer brace-quoting [...] variables'](https://google.github.io/styleguide/shell.xml?showone=Variable_expansion#Variable_expansion), but also prefers consistency so I guess if we've already got a majority un-brace-quoted it makes sense to settle on that.)

Reply:
Ah, then I must have been remembering some other guideline that said not to brace unless necessary. Will clean up in a future CL.